src/security-server/admin-service/application/src/main/java/org/niis/xroad/securityserver/restapi/config/AcmeConfig.java

```diff
@@ -60,7 +60,7 @@ public WebServerFactoryCustomizer<TomcatServletWebServerFactory> acmeChallengeCu
     private void acmeChallengeCustomizer(TomcatServletWebServerFactory factory) {
         var connector = new Connector(Http11NioProtocol.class.getName());
         connector.setScheme("http");
-        connector.setPort(80);
```

Comment:
Since the ACME challenge port can be customised now, it would be good to log it during startup so that it's possible to see from the logs what port is used.